Common/Product/SharedProject/ProcessOutput.cs

```diff
@@ -237,27 +234,18 @@ sealed class ProcessOutput : IDisposable {
         ) {
             var outFile = Path.GetTempFileName();
             var errFile = Path.GetTempFileName();
-            var psi = new ProcessStartInfo("cmd.exe");
-            psi.CreateNoWindow = true;
-            psi.WindowStyle = ProcessWindowStyle.Hidden;
-            psi.UseShellExecute = true;
-            psi.Verb = "runas";
-
-            string args;
-            if (quoteArgs) {
-                args = string.Join(" ", arguments.Where(a => a != null).Select(QuoteSingleArgument));
-            } else {
-                args = string.Join(" ", arguments.Where(a => a != null));
-            }
-            psi.Arguments = string.Format("/S /C \"{0} {1} >>{2} 2>>{3}\"",
-                QuoteSingleArgument(filename),
-                args,
-                QuoteSingleArgument(outFile),
-                QuoteSingleArgument(errFile)
-            );
-            psi.WorkingDirectory = workingDirectory;
```

Comment:
I tried testing the edge cases I could think of, but this is widely used code that is also shared with common so I'm sure I missed some areas. My understanding is that `pushd` replaces the need to set the working directory for executing the actual command. We are only relying on the initial, default  working directory to pick up `cmd.exe`.

Any specific edge cases you had in mind?